namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-dev/resources/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "team_name"                  { default = "formbuilder-dev" }
```

Comment:
Since these values do not change, you could use [`locals`](https://www.terraform.io/docs/configuration/locals.html). I'm not asking you to change them, just FYI :)